Jenkinsfile

```diff
@@ -70,20 +64,33 @@ pipeline{
 				}
 			}
 		}
+		// This stage compares the line counts of the orthopairs files generated between the current and previous release.
+		// An intelligible output should be visible at the console logs for the build.
+		stage('Post: Orthopairs file line counts') {
+		    steps {
+		        script {
+		            def releaseVersion = utils.getReleaseVersion()
+		            def previousReleaseVersion = utils.getReleaseVersion().toInteger() - 1
```

Comment:
Since `previousReleaseVersion` is a common thing we use, maybe a function could for it be added to the shared library?

Reply:
Done

Reply:
Great!  Does the 'getPreviousReleaseVersion()` function call need to be added here?  I don't see that change on my end.

Reply:
Yep! Good catch